lib/app.js

```diff
@@ -67,7 +68,11 @@ const MAPPINGS = {
   // FIXME add appId
 
   title: ['ds:3', 0, 0, 0],
-  description: ['ds:3', 0, 10, 0, 1],
+  description: {
+    path: ['ds:3', 0, 10, 0, 1],
+    fun: (text) => descriptionText(text),
```

Comment:
you don't need a lambda here, can just reference `descriptionText`

Reply:
You're right. Fixing in a moment.